abb_irb120_support/CMakeLists.txt

```diff
@@ -12,8 +13,7 @@ if (CATKIN_ENABLE_TESTING)
     DEPENDENCIES
       # not nice, but industrial_robot_client doesn't prefix its targets
       joint_trajectory_action
-      abb_driver_robot_state
-      abb_driver_motion_download_interface)
+      abb_driver_robot_state)
```

Comment:
@Levi-Armstrong is abb_driver_motion_download_interface needed anymore?  I couldn't find it anywhere.